code/ui/components/src/tabs/tabs.stories.tsx

```diff
@@ -202,7 +202,7 @@ export const StatefulDynamicWithOpenTooltip = {
     const canvas = within(canvasElement);
 
     await waitFor(async () => {
-      await expect(canvas.getAllByRole('tab')).toHaveLength(3);
+      await expect(canvas.getAllByRole('tab')).toHaveLength(4);
```

Comment:
This will still remain flaky. Rather change the viewport in line 199 to 380px. Then a couple of pixels width difference doesn`t matter anymore.

Reply:
It seems, that chromatic unfortunately doesn‘t take the same viewport width every time, but it is sometimes shifted by a couple of pixels.

Reply:
Please adjust it also in line 233

Reply:
ok

Reply:
If you change the number from 4 back to 3 it'll be all good 👌